### ISSUE TITLE
Upgrade OpenPGP.js

### ISSUE THREAD

**Issue by rugk** [Gaff/pgp.help#31]
AFAIK you still use OpenPGP.js v1.2, but there is even [v2](https://github.com/openpgpjs/openpgpjs) available now, so what about upgrading?

**Comment by Gaff** [comment]
Hmm open pgp 1.2 changes the API a little bit so it's non-trivial. I've upgraded to 1.5.7.

**Comment by rugk** [comment]
Yeah, that's nice anyway. Still an upgrade to 2.0 might be considered for the future.

**Comment by Gaff** [comment]
Definitely. I'd also like to give it a few months to bed in. Don't want to be tripped up by any security issues.

**Comment by Gaff** [comment]
Ok it would be. I'm kinda willing to work on this if I have any spare cycles.

**Comment by Gaff** [comment]
OK done :D
